include/SDL3/SDL_surface.h

```diff
@@ -94,12 +94,11 @@ typedef enum SDL_ScaleMode
  *
  * \since This enum is available since SDL 3.2.0.
  */
-typedef enum SDL_FlipMode
-{
-    SDL_FLIP_NONE,          /**< Do not flip */
-    SDL_FLIP_HORIZONTAL,    /**< flip horizontally */
-    SDL_FLIP_VERTICAL       /**< flip vertically */
-} SDL_FlipMode;
+typedef Uint32 SDL_FlipMode;
```

Comment:
This is potentially an ABI break. We guarantee that enums are the size of an int, but an int may not necessarily be the size of a Uint32. I think it is for all currently supported platforms, so maybe this is okay?

@icculus, thoughts?